### PR TITLE
Host name check now ignores case

### PR DESCRIPTION
**Pull request by JoachimL**
We encountered an issue where a hostname of "Sitemap.xml" broke
our site at runtime, since the system expected "sitemap.xml".

This fix makes that check ignore case. It also adds tests to prove the
fix. In order to make the method testable, it was made public
(this seemed like the least intrusive way of fixing it, and
it is a static method without side effects, so it being public
should not be an issue.

I unfortunately do not have VS 2013 on my dev machine at the moment, so I've also updated the solution file to VS2015 format.

### CODE REVIEW

File: .vs/config/applicationhost.config [hunk 1]
```diff
@@ -0,0 +1,1031 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (fatso83) [inline .vs/config/applicationhost.config]: `.vs` should be put in `.gitconfig`. It is user dependent, like `*suo` and `*.user` files.

Comment (fatso83) [inline .vs/config/applicationhost.config]: Made #42 to address this.

**Comment by frederikvig** [comment]
@JoachimL could you please update this PR with the latest changes?

**Comment by JoachimL** [comment]
Yes, will do ASAP.

**Comment by fatso83** [comment]
@JoachimL: repeating myself, but as previously mentioned you did a lot of the the same things as I did in #39, so you will probably have to implement the same changes as well, including moving and renaming the unit test directory. But there already is a [unit test project](https://github.com/Geta/SEO.Sitemaps/blob/master/test/Geta.SEO.Sitemaps.Tests/CompressionHandlerTest.cs), so you might as well just add your test file [to the `test/Geta.SEO.Sitemaps.Tests/` project](https://github.com/Geta/SEO.Sitemaps/blob/master/test/Geta.SEO.Sitemaps.Tests/), rather than wait for yet another round where you'll probably be told the same 😃 

The end diff will be quite a bit smaller too (one new file and two modified files).

**Comment by JoachimL** [comment]
Right. Will give it another go.

**Comment by frederikvig** [comment]
Hi @JoachimL any updates on this?

**Pull request closed without merging**